package.json

```diff
@@ -20,7 +20,7 @@
     "test": "npm run test:lint && npm run test:unit",
     "test:lint": "standard",
     "test:lint:fix": "standard --fix",
-    "test:unit": "node test/parallel.mjs && node test/message.js && node test/node-core-test.js"
+    "test:unit": "node test/parallel.mjs && node test/node-core-test.js"
```

Comment:
why?

Reply:
See the OP:

> This also disables the `test/message` because they rely heavily on `AbortSignal` support.

Reply:
I think there are two files that can be skipped on node 14/16 (abort and abort suite), to make message tests work

Reply:
Ah yes maybe it's worth simply removing those two files then, let me take a look.